src/popover/popover.js

```diff
@@ -16,4 +16,25 @@ angular.module( 'ui.bootstrap.popover', [ 'ui.bootstrap.tooltip' ] )
 
 .directive( 'popover', [ '$tooltip', function ( $tooltip ) {
   return $tooltip( 'popover', 'popover', 'click' );
+}])
+
+.directive( 'popoverTemplatePopup', [ function () {
+  return {
+    restrict: 'EA',
+    replace: true,
+    scope: { title: '@', template: '=', placement: '@', animation: '&', isOpen: '&' },
```

Comment:
why not use 1-way binding instead of text binding so `title` and `placement` can also be updated?